app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -798,7 +798,7 @@ def build_toolbar_hide_button(id)
       when "miq_template_refresh"
         return true if @record && !@record.ext_management_system && !(@record.host && @record.host.vmm_product.downcase == "workstation")
       when "miq_template_scan", "image_scan"
-        return true unless @record.is_available?(:smartstate_analysis) || @record.is_available_now_error_message(:smartstate_analysis)
+        return true unless @record.supports_smartstate_analysis? || @record.unsupported_reason(:smartstate_analysis)
```

Comment:
you can also remove the call to `unsupported_reason` here, because checking for support does not rely on the presence of a reason - and should not have been in the first place